Documentation/k8s-pre-reqs.md

```diff
@@ -22,6 +22,37 @@ Creating the Rook operator requires privileges for setting up RBAC. To launch th
 -subj "/CN=admin/O=system:masters"
 ```
 
+## Flexvolume Configuration
+
+Rook uses [Flexvolume](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md) to integrate with Kubernetes for performing storage operations. In some operating systems where Kubernetes is deployed, the [default Flexvolume plugin directory](https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md#prerequisites) (the directory where flexvolume drivers are installed) is **read-only**.
```

Comment:
I personally would change "operating systems" to "environments. Because some people use ContainerLinux without running kubelet inside a container but with direct binaries.